### PR TITLE
update unittest and neon  for neo3

### PR DESCRIPTION
**Pull request by lightszero**
1.fix appcall -> neo3 style syscall
2.fix devpack's checksig opcode -> neo3 syscall.
3.change unittest's NEOVM  -> neo3

### CODE REVIEW

File: Neo.Compiler.MSIL.UnitTests/Neo.Compiler.MSIL.UnitTests.csproj [hunk 1]
```diff
@@ -7,13 +7,33 @@
...
+    </None>
+    <None Update="needlib\mscorlib.ni.dll">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+    <None Update="needlib\mscorlib.resources.dll">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+    <None Update="needlib\System.dll">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
```
Comment (shargon) [inline Neo.Compiler.MSIL.UnitTests/Neo.Compiler.MSIL.UnitTests.csproj]: This library should be provided by dotnet, isn't it?

File: Neo.SmartContract.Framework/OpCode.cs [hunk 2]
```diff
@@ -3,141 +3,723 @@
     public enum OpCode : byte
     {
         // Constants
-        PUSH0 = 0x00, // An empty array of bytes is pushed onto the stack.
```
Comment (shargon) [inline Neo.SmartContract.Framework/OpCode.cs]: we should use the enum from neoVM

File: neo-devpack-dotnet.sln [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.28803.452
+# Visual Studio 15
```
Comment (shargon) [inline neo-devpack-dotnet.sln]: downgrade?

**Comment by erikzhang** [comment]
There are compiler errors.

File: Neo.Compiler.MSIL.UnitTests/Neo.Compiler.MSIL.UnitTests.csproj (hunk 1 above)
Comment (lightszero) [reply Neo.Compiler.MSIL.UnitTests/Neo.Compiler.MSIL.UnitTests.csproj]: no , i will use rosyln to build untitest, that file is for rosyln.

File: Neo.SmartContract.Framework/OpCode.cs (hunk 2 above)
Comment (lightszero) [reply Neo.SmartContract.Framework/OpCode.cs]: it is copy from neoVM

File: neo-devpack-dotnet.sln (hunk 3 above)
Comment (lightszero) [reply neo-devpack-dotnet.sln]: Visual Studio 15 = vs2017
16=vs2019

Are you guys all use vs2019?

Comment (erikzhang) [reply neo-devpack-dotnet.sln]: All 2019 😁

**Comment by erikzhang** [comment]
@lightszero Please check #45

File: Neo.SmartContract.Framework/OpCode.cs (hunk 2 above)
Comment (shargon) [reply Neo.SmartContract.Framework/OpCode.cs]: why we don't use the same enun?

**Comment by lightszero** [comment]
most part is in #45,so close this.

**Pull request closed without merging**